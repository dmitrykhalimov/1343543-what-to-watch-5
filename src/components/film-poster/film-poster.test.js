import React from "react";
import renderer from "react-test-renderer";
import FilmPoster from "./film-poster";

describe(`FilmPoster`, () => {
  it(`Should FilmPoster render correctly`, () => {
    const tree = renderer
      .create(
          <FilmPoster
            title={``}
            poster={``}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

