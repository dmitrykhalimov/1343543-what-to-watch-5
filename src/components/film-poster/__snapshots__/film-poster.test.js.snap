// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmPoster Should FilmPoster render correctly 1`] = `
<div
  className="movie-card__poster"
>
  <img
    alt=""
    height="327"
    src=""
    width="218"
  />
</div>
`;
